tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -123,14 +130,14 @@ execute: |
 
     # Test the unsealed key
     echo "Ensure that we can open the encrypted device using the unsealed key"
```

Comment:
As a followup we should probably check the same with for ubuntu-save with save.key and reinstall.key.

Reply:
I'll open a followup PR with this and a tweak for NOMATCH that @sergiocazzolato pointed out.